public/index.php

```diff
@@ -5,13 +5,15 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge" />
   <?php
   // update these fields
+  // assets_path is after projects.statesman.com to root of project
   $meta = array(
+    "url" => "http://projects.statesman.com/news/single-page-project/",
+    "assets_path" => "assets/",
     "title" => "Single page project | Statesman.com",
     "description" => "Description for single-page-project.",
-    "thumbnail" => "http://projects.statesman.com/site_path/assets/share.jpg",
-    "shortcut_icon" => "http://media.cmgdigital.com/shared/media/2015-08-28-16-58-55/web/site/www_mystatesman_com/images/favicon.ico",
-    "apple_touch_icon" => "http://media.cmgdigital.com/shared/theme-assets/242014/www.statesman.com_fa2d2d6e73614535b997734c7e7d2287.png",
-    "url" => "http://projects.statesman.com/news/single-page-project/",
+    "thumbnail" => "share.jpg",
+    "shortcut_icon" => "favicon.ico",
+    "apple_touch_icon" => "favicon-apple-touch-icon.png",
```

Comment:
http://media.cmgdigital.com/shared/theme-assets/242014/www.statesman.com_fa2d2d6e73614535b997734c7e7d2287.png is blue